dimagi/pages/templates/sections/pillar/mobile_data_collection/chapter7/part3b.html

```diff
@@ -113,7 +113,7 @@ <h5>
     </p>
 
     <p class="align-center pad-top-small pad-bottom-medium">
-      <a href="https://dimagi.com/blog/mobile-data-collection-implement-and-train/"
+      <a href="https://dimagi.com/blog/mobile-data-collection-app-deploy"
```

Comment:
nitpick. it's best practice to end links with `/`

Reply:
@biyeun added nitpick and thanks for the comment.